submissions/josephOlasimbo/javascript/conditionals/index.js

```diff
@@ -0,0 +1,19 @@
+let value;
+
+for (let i = 1; i <= 116; i++) {
```

Comment:
116 can be anything, but the question says for the positive difference between 7 - 123. This code doesn't show any relationship to 7 or 123